Package.swift

```diff
@@ -33,6 +33,14 @@ let package = Package(
       name: "SwiftFormatConfiguration",
       targets: ["SwiftFormatConfiguration"]
     ),
+    .plugin(
+      name: "Format Source Code",
```

Comment:
I think we should remove the spaces from the _target_ names and subdirectory names. Names like `FormatPlugin` and `LintPlugin` would align well with the naming we've used elsewhere.

For the `name` argument of the `.plugin`, I'm less sure. I believe this surfaces in the Xcode UI? Is there anywhere else that this string is used (like a command line) where spaces would cause problems?

Reply:
I fixed with reference to [swift-argument-parser](https://github.com/apple/swift-argument-parser/blob/76259ebe8c477ecc4e938d0831aea4c25f9ee3ee/Package%40swift-5.6.swift)

[zunda-pixel/swift-format/pull/1](https://github.com/zunda-pixel/swift-format/pull/1)

Xcode  uses [Package.targets.plugin.name](https://github.com/zunda-pixel/swift-format/blob/41cf80954c9ab0bc8b3a562d89ad788534515649/Package.swift#L103) as UI

<img width="300" src="https://user-images.githubusercontent.com/47569369/200998494-40f5e720-5a31-4fe6-a503-c633ba266d25.png">